splinterd/api/static/openapi.yml

```diff
@@ -581,8 +581,8 @@ paths:
                 $ref: '#/components/schemas/Error'
 
   /scabbard/{circuit}/{service_id}/batch_statuses:
-    post:
-      description: Send a list of Sabre batches to the specified Scabbard service
+    get:
+      description: Fetches the committed statuses for a set of Sabre batches submitted
```

Comment:
I would remove "committed" from this description, because it seems to imply that it only gets status when they are committed. Also, why "submitted" at the end of the description? Same for response description below.

Reply:
Addressed.